### PR TITLE
Mount secret for kam test user

### PR DESCRIPTION
**Pull request by amitkrout**
Mount secret for kam test user

### CODE REVIEW

File: ci-operator/config/redhat-developer/kam/redhat-developer-kam-master__v4.5.yaml [hunk 1]
```diff
@@ -31,6 +31,10 @@ tests:
     test:
     - as: integration-e2e-steps
       commands: scripts/openshiftci-presubmit-all-tests.sh
+      credentials:
+      - mount_path: /var/run/kam-data
+        name: kam-user-secret
+        namespace: ci
```
Comment (droslean) [inline ci-operator/config/redhat-developer/kam/redhat-developer-kam-master__v4.5.yaml:37]: you should remove this line.

**Comment by amitkrout** [comment]
pr https://github.com/openshift/release/pull/13202 has been merged
/hold cancel

**Comment by amitkrout** [comment]
$ oc get secret -n kam
NAME                       TYPE                                  DATA   AGE
[...]
kam-user-secret            Opaque                                1      6d
quay-docker-config-auth    Opaque                                1      6s

**Comment by amitkrout** [comment]
ping @wtam2018 @sbose78

**Review by wtam2018 — COMMENTED** [review]
/lgtm

**Pull request merged**